src/tests/utils/constants.cairo

```diff
@@ -6,103 +6,103 @@ use starknet::class_hash::class_hash_const;
 use starknet::contract_address_const;
 use starknet::secp256_trait::Secp256Trait;
 
-pub(crate) const DECIMALS: u8 = 18_u8;
-pub(crate) const SUPPLY: u256 = 2000;
-pub(crate) const VALUE: u256 = 300;
-pub(crate) const FELT_VALUE: felt252 = 'FELT_VALUE';
-pub(crate) const ROLE: felt252 = 'ROLE';
-pub(crate) const OTHER_ROLE: felt252 = 'OTHER_ROLE';
-pub(crate) const TOKEN_ID: u256 = 21;
-pub(crate) const TOKEN_ID_2: u256 = 121;
-pub(crate) const TOKEN_VALUE: u256 = 42;
-pub(crate) const TOKEN_VALUE_2: u256 = 142;
-pub(crate) const PUBKEY: felt252 = 'PUBKEY';
-pub(crate) const NEW_PUBKEY: felt252 =
-    0x26da8d11938b76025862be14fdb8b28438827f73e75e86f7bfa38b196951fa7;
-pub(crate) const DAPP_NAME: felt252 = 'DAPP_NAME';
-pub(crate) const DAPP_VERSION: felt252 = 'DAPP_VERSION';
-pub(crate) const SALT: felt252 = 'SALT';
-pub(crate) const SUCCESS: felt252 = 123123;
-pub(crate) const FAILURE: felt252 = 456456;
-pub(crate) const MIN_TRANSACTION_VERSION: felt252 = 1;
+pub const DECIMALS: u8 = 18_u8;
+pub const SUPPLY: u256 = 2000;
+pub const VALUE: u256 = 300;
+pub const FELT_VALUE: felt252 = 'FELT_VALUE';
+pub const ROLE: felt252 = 'ROLE';
+pub const OTHER_ROLE: felt252 = 'OTHER_ROLE';
+pub const TOKEN_ID: u256 = 21;
+pub const TOKEN_ID_2: u256 = 121;
+pub const TOKEN_VALUE: u256 = 42;
+pub const TOKEN_VALUE_2: u256 = 142;
+pub const PUBKEY: felt252 = 'PUBKEY';
```

Comment:
They are not public, but the crate annotation here was redundant since making the constants module public only for the crate does the work already.

Reply:
I get the idea, but I think this seems a bit odd. When I see `pub`, I expect the item to be public, no? It's removing the explicit visibility and I would expect that we'd get questions on why this isn't visible despite the `pub` annotation. If we use this pattern with all of our crate-specific items, I believe this would make the library more difficult to maintain and read. Do you disagree?

And with that said, I'm wondering if we'll have to make these public anyway for when we migrate to workspaces

Reply:
I agree that it may be misleading for someone. And if we decide to employ this approach (without declaring `crate` visibility repeatedly), we should stick to it in all the modules for consistency

Reply:
Let's keep it then, and maybe revisit the discussion in the future.